lib/ohai/plugins/linux/network.rb

```diff
@@ -384,6 +384,14 @@ def favored_default_route(routes, iface, default_route, family)
     counters Mash.new unless counters
     counters[:network] = Mash.new unless counters[:network]
 
+    # ohai.plugin[:network][:default_route_table] = 'default'
+    if configuration(:default_route_table).nil? || configuration(:default_route_table).empty?
+      default_route_table = 'main'
```

Comment:
Great! In that case, it seems reasonable to make the "new" default be `table main` :)